src/index.js

```diff
@@ -17,6 +17,10 @@ import CombinedFacets from './modules/combinedFacets';
 import ViewTypes from './modules/viewTypes';
 import UnbxdSearchWrapper from './UnbxdSearchWrapper';
 
+if(window.localStorage && window.localStorage.getItem("debug")) {
+    console.log(UnbxdSearchWrapper)
```

Comment:
can we remove this console statement?

Reply:
added this for debugging it in clients production bundle. prints only if we add debug in localstorage